src/setup/cortx_deploy

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+DEFAULT_LOG_LEVEL=0
+function log {
+    LOG_LEVEL=${DEBUG:-$DEFAULT_LOG_LEVEL}
+    [[ $LEVEL -lt $LOG_LEVEL ]] && return
```

Comment:
LEVEL should be taken from $1
LEVEL=$1
LOG_LEVEL should come from user.

Reply:
done